skimage/data/tests/test_data.py

```diff
@@ -140,6 +144,11 @@ def test_lfw_subset():
     """ Test that "lfw_subset" can be loaded."""
     data.lfw_subset()
 
+@skipif(not has_pooch, reason="pooch is not installed")
```

Comment:
Can be addressed elsewhere, but if we need to skip if pooch is not installed, we must be consistent for the other online contents.

Reply:
indeed this was not required, I removed the skip.